### PR TITLE
Typescript

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "license": "BSD-3-Clause",
   "repository": "Polymer/pwa-starter-kit-hn",
   "scripts": {
-    "start": "polymer serve",
-    "build": "npm run build:prpl-server && npm run build:static",
+    "start": "tsc --watch & polymer serve",
```
Comment (frankiefu) [inline package.json]: Just a note that `&` doesn't work on windows. We can replace it with whatever we decided on [here](https://github.com/Polymer/pwa-starter-kit/pull/244)

File: src/components/hn-app.ts [hunk 2]
```diff
@@ -38,11 +38,8 @@ export class HnAppElement extends connect(store)(LitElement) {
     </div>`;
   }
 
-  static get properties() {
-    return {
-      _view: { type: String }
-    };
-  }
+  @property()
+  _view: string|undefined;
```
Comment (frankiefu) [inline src/components/hn-app.ts]: Add `private` if you want stricter typing.

File: src/components/hn-loading-button.ts [hunk 3]
```diff
@@ -48,11 +48,8 @@ export class HnLoadingButton extends LitElement {
     </button>`;
   }
 
-  static get properties() {
-    return {
-      loading: { type: Boolean }
-    };
-  }
+  @property()
```
Comment (frankiefu) [inline src/components/hn-loading-button.ts]: `@property({type: Boolean})` to let LitElement knows the type.

File: src/components/hn-app.ts (hunk 2 above)
Comment (frankiefu) [reply src/components/hn-app.ts]: FYI, I think:
`_view?: string;` is the same as `_view: string|undefined;`

File: src/components/hn-loading-button.ts (hunk 3 above)
Comment (keanulee) [reply src/components/hn-loading-button.ts]: I don't need that if I'm not setting by attribute, right? (only `<hn-loading-button .loading=${val}>`)

Comment (frankiefu) [reply src/components/hn-loading-button.ts]: This is true. This is mainly for consistency, typescript type to be same as property type.

**Pull request merged**